modules/tinymce/src/core/main/ts/delete/BlockRangeDelete.ts

```diff
@@ -58,14 +57,44 @@ const emptyEditor = (editor: Editor): Optional<() => void> => {
   });
 };
 
+const getClosestEditableDiv = (editor: Editor): HTMLElement | null => {
+  const node = editor.selection.getNode();
+  return node.closest('.editable');
```

Comment:
Ještě prosím místo natvrdo napsané classy použít 
```suggestion
  return node.closest(`.${editor.getParam('editable_class')}`);
```